event-based-simulators/extras/ExportProfileData.py

```diff
@@ -0,0 +1,233 @@
+    response = requests.get(f'{C8Y_BASE}/inventory/managedObjects/{DEVICE_ID}',
+                            headers=C8Y_HEADERS)
+    if not response.ok:
+        logger.error(
+            f"Connection to url '{C8Y_BASE}/inventory/managedObjects/{DEVICE_ID}' failed. Check your parameters in environment file again")
+        sys.exit()
+    else:
+        try:
+            deviceName = response.json()['name']
+        except:
+            logger.error(f"Device #{DEVICE_ID} does not have name")
+            sys.exit()
+
+    return deviceName
+
+
+def listAlarms(c8y, device, createFrom, createTo):
+    jsonAlarmsList = []
+    # Create a count variable as a json/dict key to save json data
+    count = 0
+    for alarm in c8y.alarms.select(source=device.id, created_after=createFrom, created_before=createTo):
+        logger.info(
+            f"Found alarm id #{alarm.id}, severity: {alarm.severity}, time: {alarm.time}, creation time: {alarm.creation_time}, update time : {alarm.updated_time}\n")
+        count += 1
+        jsonAlarmsList.append(alarm.to_json())
+    return jsonAlarmsList
+
+
+def listMeasurements(c8y, device, createFrom, createTo):
+    jsonMeasurementsList = []
+    # Create a count variable as a json/dict key to save json data
+    count = 0
+    for measurement in c8y.measurements.select(source=device.id, after=createFrom, before=createTo):
+        logger.info(f"Found measurement id #{measurement.id}\n")
+        count += 1
+        jsonMeasurementsList.append(measurement.to_json())
+    return jsonMeasurementsList
+
+
+def appendDataToJsonFile(jsonDataList, filePath, data_type, json_data={}):
+    # Create new json file or add data to an existing json file
+    with open(filePath, 'w') as f:
+        json_data[f"{data_type.capitalize()}"] = jsonDataList
+        json.dump(json_data, f, indent=2)
+
+
+def getExternalIdReponse(deviceId):
+    C8Y_BASE = removeSlashFromBaseUrl()
+    externalIdResponse = requests.get(f'{C8Y_BASE}/identity/globalIds/{deviceId}/externalIds',
+                                      headers=C8Y_HEADERS)
+    if not externalIdResponse.ok:
+        logger.error(
+            f"Connection to url '{C8Y_BASE}/identity/globalIds/{deviceId}/externalIds' failed. Check your parameters in environment file again")
+        sys.exit()
+    else:
+        return externalIdResponse
+
+
+def checkDeviceExternalIdById(deviceId):
+    externalIdResponse = getExternalIdReponse(deviceId)
+
+    try:
+        deviceExternalId = externalIdResponse.json()['externalIds'][0]['externalId']
+        deviceExternalIdType = externalIdResponse.json()['externalIds'][0]['type']
+    except:
+        logger.error(f"Could not find external id for the device with id {deviceId}")
+        print(f"Could not find external id for the device with id {deviceId}")
+        return None, None
+
+    return deviceExternalId, deviceExternalIdType
+
+
+def isExternalIdHasEventBasedSimulatorProfileType(deviceExternalIdType):
```

Comment:
rename: ExternalIdHasProfileType?

Reply:
since the function is a boolean one, and also it is used to only check if the External Id Type matches with the type c8y_EventBasedSimulatorProfile, I decide to name it like that so it can show clearly what the purpose of the function is.